airbyte-cdk/python/airbyte_cdk/models/airbyte_protocol.py

```diff
@@ -117,7 +117,7 @@ class Config:
     supportsNormalization: Optional[bool] = Field(False, description="If the connector supports normalization or not.")
     supportsDBT: Optional[bool] = Field(False, description="If the connector supports DBT or not.")
     supported_destination_sync_modes: Optional[List[DestinationSyncMode]] = Field(
-        None, description="List of destination sync modes supported by the connector"
+        [], description="List of destination sync modes supported by the connector"
```

Comment:
this is auto generated so it will be overwritten when the build runs. Why are you looking to make this change?